src/main/java/com/commercetools/sync/producttypes/utils/ProductTypeUpdatePlainEnumActionUtils.java

```diff
@@ -0,0 +1,136 @@
+     *
+     * <p>If the list of new {@link EnumValue}s is {@code null}, then remove actions are built for
+     * every existing plain enum value in the {@code oldEnumValues} list.
+     *
+     * @param attributeDefinitionName the attribute name whose plain enum values are going to be synced.
+     * @param oldEnumValues           the old list of plain enum values.
+     * @param newEnumValues           the new list of plain enum values.
+     * @return a list of plain enum values update actions if the list of plain enum values is not identical.
+     *         Otherwise, if the plain enum values are identical, an empty list is returned.
+     * @throws DuplicateKeyException in case there are plain enum values with duplicate keys.
+     */
+    @Nonnull
+    public static List<UpdateAction<ProductType>> buildEnumValuesUpdateActions(
+        @Nonnull final String attributeDefinitionName,
+        @Nonnull final List<EnumValue> oldEnumValues,
+        @Nullable final List<EnumValue> newEnumValues)
+        throws DuplicateKeyException {
+
+        if (newEnumValues != null && !newEnumValues.isEmpty()) {
+            return buildUpdateActions(
+                attributeDefinitionName,
+                oldEnumValues,
+                newEnumValues
+            );
+        } else {
+            return buildRemoveEnumValuesUpdateActions(
+                attributeDefinitionName,
+                oldEnumValues,
+                newEnumValues
+            )
+                .map(Collections::singletonList)
+                .orElse(emptyList());
+        }
+    }
+
+
+    /**
+     * Compares a list of old {@link EnumValue}s with a list of new {@link EnumValue}s for a given attribute
+     * definition.
+     * The method serves as a generic implementation for plain enum values syncing. The method takes in functions
+     * for building the required update actions (AddEnumValue, RemoveEnumValue, ChangeEnumValueOrder and 1-1
+     * update actions on plain enum values (e.g. changeLabel) for the required resource.
+     *
+     * @param attributeDefinitionName the attribute name whose plain enum values are going to be synced.
+     * @param oldEnumValues           the old list of plain enum values.
+     * @param newEnumValues           the new list of plain enum values.
+     * @return a list of plain enum values update actions if the list of plain enum values is not identical.
+     *         Otherwise, if the plain enum values are identical, an empty list is returned.
+     * @throws DuplicateKeyException in case there are plain enum values with duplicate keys.
+     */
+    @Nonnull
+    private static List<UpdateAction<ProductType>> buildUpdateActions(
+        @Nonnull final String attributeDefinitionName,
+        @Nonnull final List<EnumValue> oldEnumValues,
+        @Nonnull final List<EnumValue> newEnumValues)
+        throws DuplicateKeyException {
```

Comment:
The `throws DuplicateKeyException` here is redundant since it's a runtime exception

Reply:
c3d86f78c7672ae9811d17d51c2cc8fc3d2d5a13